bin/run-kibana.sh

```diff
@@ -24,34 +24,21 @@ erb -T 2 -r uri -r base64 ./kibana.erb > /etc/nginx/sites-enabled/kibana || {
   exit 1
 }
 
-# If we don't have a version set, then try to guess one form the Elasticsearch server.
-if [[ -z "$KIBANA_ACTIVE_VERSION" ]]; then
-  KIBANA_VERSION_PARSER="
-  require 'json'
-  version = JSON.parse(STDIN.read)['version']['number']
-  print 41 if version.start_with?('1.')
-  print 44 if version.start_with?('2.')
-  print 5 if version.start_with?('5.')"
-  KIBANA_ACTIVE_VERSION="$(curl "$DATABASE_URL" 2>/dev/null | ruby -e "$KIBANA_VERSION_PARSER" 2>/dev/null)"
-fi
-
-# If we still don't have a version, fall back to the default.
-if [[ -z "$KIBANA_ACTIVE_VERSION" ]]; then
-    echo "Warning: unable to fetch Elasticsearch version, and none configured. Defaulting to 5.x. Consider setting KIBANA_ACTIVE_VERSION."
-    KIBANA_ACTIVE_VERSION="5"
-fi
-
-echo "KIBANA_ACTIVE_VERSION is set to: '$KIBANA_ACTIVE_VERSION'"
-
-KIBANA_VERSION_PTR="KIBANA_${KIBANA_ACTIVE_VERSION}_VERSION"
-KIBANA_VERSION="${!KIBANA_VERSION_PTR}"
-
 # Run config
-erb -T 2 -r uri "/opt/kibana-${KIBANA_VERSION}/config/kibana.yml.erb" > "/opt/kibana-${KIBANA_VERSION}/config/kibana.yml" || {
+erb -T 2 -r uri -r base64 "/opt/kibana/config/kibana.yml.erb" > "/opt/kibana/config/kibana.yml" || {
   echo "Error creating kibana config file"
   exit 1
 }
 
+# Don't bother checking ES version before startup in testing, some tests don't actually start ES
+if [[ -z "$TESTING" ]]; then
+  bash ./check-es-version.sh || {
```

Comment:
- Ensure the script is executable and has a shebang, and remove the `bash` call here
- I'm not a fan of expecting `check-es-version.sh` to be in the current directory 

Besides, consider using `if ! ...` as opposed to `|| { }` to make the intent a little clearer here